dhis-2/dhis-api/src/main/java/org/hisp/dhis/note/TrackedEntityCommentService.java

```diff
@@ -25,28 +25,30 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.hisp.dhis.trackedentitycomment;
+package org.hisp.dhis.note;
 
 /**
  * @author Chau Thu Tran
+ * @deprecated use {@link org.hisp.dhis.tracker.note} service instead
  */
+@Deprecated(since = "2.41")
```

Comment:
I don't think we are there yet in our code organization to deprecate this one.
We have similar structure for enrollments and events and the service interface is in `dhis-api` and the implementation in `dhis-service-core`. We still need it to be outside `dhis-service-tracker` as it is needed in other parts of the system (deletion handlers for examples), and they are not depending on tracker.

Reply:
ok i've reverted the changes. However, currently the deletion handler for notes is not in triggered, we'll see how to deal with that with https://dhis2.atlassian.net/browse/DHIS2-15806